app/components/conferences/admin/editEvent.js

```diff
@@ -0,0 +1,168 @@
+import { useEffect, useState } from "react";
```

Comment:
Please rename to `EditEvent.js`, currently the components are named starting in caps. And helpers start with small caps.
Thank you!